ashlar/scripts/make_alignment_movie.py

```diff
@@ -67,5 +68,6 @@ def main(args):
     subprocess.call(CMD.split(' '))
     return 0
 
+
 if __name__ == '__main__':
     sys.exit(main(sys.argv[1:]))
```

Comment:
This conflicts with the default value for `argv` in `main`. Should drop the `[1:]` to be consistent, or just drop the arg entirely.